zabbix_auto_config/processing.py

```diff
@@ -259,7 +259,7 @@ def merge_hosts(self, hostname):
         merged_host = {
             "enabled": any([host["enabled"] for host in hosts]),
             "hostname": hostname,
-            "inventory": None,  # TODO
+            "inventory": dict(itertools.chain.from_iterable(host["inventory"].items() for host in hosts if "inventory" in host)),
```

Comment:
@mustafaocak and I spoke about this outside of the PR. Properties will silently be lost here if multiple sources set the same inventory property.

How about handling this with a bit more "verbose" code (like interfaces, importance and proxy_pattern below).

I'm not sure if this is the best solution, but we will at least get a warning when this "property loss" occurs:
```python
inventory = hosts[0]["inventory"] if "inventory" in hosts[0] else {}
for host in hosts[1:]:
    if "inventory" in host:
        for k, v in host["inventory"].items():
            if k in inventory and v != inventory[k]:
                logging.warning("Same inventory ('%s') set multiple times for host: '%s'", k, hostname)
            else:
                inventory[k] = v
```